webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
It would be cool to add `webpack-dev-server`. We will need it for development. https://webpack.js.org/configuration/dev-server/